proxies/live/apiproxy/policies/AssignMessage.SetEchoTruststore.xml

```diff
@@ -0,0 +1,6 @@
+<AssignMessage name="AssignMessage.SetEchoTruststore">
+    <AssignVariable>
+        <Name>truststore</Name>
+        <Value/>
+    </AssignVariable>
+</AssignMessage>
```

Comment:
This is the same as ClearTruststore.
Is there really no Echo Truststore? if not using ClearTruststore in the taget.xml instead would be more readable

Reply:
Ah yup good spot, ClearTruststore is a left over of some logic I had in the proxy previously. Due to us using our own CA to sign our FT environments we need to set a separate CA when calling the FT environments, however this truststore does not trust the echo target. This logic is setting the truststore to be an empty value to signal to apigee that it should use the default CA when calling the echo target rather than using our internal CA.

Happy to swap over to using the ClearTruststore policy instead, but I was a bit worried that it would be a bit confusing as the truststore variable won't exist until this policy is called, as this policy creates the truststore flow variable as an empty value.

Reply:
Can we put a comment in along the lines of "setting empty truststore  to signal to apigee that it should use the default.
It was the only bit in the code where I didn't get it immediately

Reply:
Added this comment in now and removed `AssignMessage.SetEchoTruststore.xml`